schema/proto3/dhcp.proto

```diff
@@ -28,11 +28,12 @@ message DHCPConfiguration {
     uint32 revision_number = 2;
 
     string request_id = 3;
-    string subnet_id = 4;
-    string mac_address = 5;
-    string ipv4_address = 6;
-    string ipv6_address = 7;
-    string port_host_name = 8; // for local DNS response
+    MessageType message_type = 4; // DELTA (default) or FULL
```

Comment:
Do we differentiate message_type for DHCP, or everything is FULL for DHCP?

Reply:
I see two possible usage of message_type for DHCP:
1. To update the repeated extra_dhcp_options or dns_entry_list in the future
2. When the DHCP resource is out of order/error, it will look for a FULL message_type to reset

We may go with everything FULL approach for DHCP, but it will have less flexability and not consistant with other resource types.

Reply:
Good thought and thank you.

Reply:
we need PM to provide the api then

Reply:
DHCP configuration is generated by DPM, if additional information is needed PM should provide it based on the contract between DPM<-> PM.

Reply:
@chenpiaoping what do you think about "PM should provide it based on the contract between DPM<-> PM"